images/hub/unfrozen/requirements.txt

```diff
@@ -0,0 +1,33 @@
+# This file is the input to requirements.txt, which is a frozen version of this.
+#
+# To update:
+# - the jupyterhub version or the frozen requirements.txt file, use the
+#   "Run workflow" button at https://github.com/jupyterhub/zero-to-jupyterhub-k8s/actions/workflows/watch-dependencies.yaml.
+# - the major version pins in this file, trigger dependabot to run via
+#   https://github.com/jupyterhub/zero-to-jupyterhub-k8s/network/updates
+#
+
+# JupyterHub itself
+jupyterhub==5.2.1
+
+# JupyterHub Spawner, kubernetes specific
+jupyterhub-kubespawner==7.*
+
+# JupyterHub Authenticator choices
+jupyterhub-firstuseauthenticator==1.*
+jupyterhub-hmacauthenticator==1.*
+jupyterhub-ldapauthenticator==2.*
+jupyterhub-ltiauthenticator==1.*
+jupyterhub-nativeauthenticator==1.*
+jupyterhub-tmpauthenticator==1.*
+oauthenticator[googlegroups,mediawiki]==17.*
+
+# JupyterHub service shutting servers after a period of inactivity
+jupyterhub-idle-culler==1.*
+
+# Other optional dependencies for additional features
+pymysql==1.*  # mysql
```

Comment:
fwiw, I wouldn't consider a bump of any of these to be a breaking change, but that's not a big deal. Perhaps they shouldn't be pinned?

Reply:
Yeah hmmm I agree, they aren't meant to be a breaking change for the user.

It could be relevant to be given a chance to read the changelog before merging though. I'm _fine with removing pins or retaining them_, but I agree they are reasonably not to be listed as breaking in the end.

I think for all major bumps here, some may not be breaking in the end, so generally a manual consideration about breaking label or not this is probably needed.